### ISSUE TITLE
Feature Query: Megadrive / SMS support plans?

### ISSUE THREAD

**Issue by Xalior** [willyvmm/mouSTer#14]
Do you have any plans to add MegaDrive / Sega Master System support?  I'm keen to try this on the Spectrum Next, which has native support for this controller type (as an extension of Kempston, logically enough) and wondered if it was on your radar?

**Comment by willyvmm** [comment]
MD has another pinout, so it's not straight forward. It requires modification on the PCB layout. 
Answering your question: YES it is planned but i need more feedback if there is someone who would like to buy it.

**Comment by Echtze1t** [comment]
> MD has another pinout, so it's not straight forward. It requires modification on the PCB layout. Answering your question: YES it is planned but i need more feedback if there is someone who would like to buy it.

Hi, I would like to help you in any way possible to get the Mouster working with the Sega Mega Drive. Please let me know what I can do to help.

**Comment by smf-** [comment]
I got a spectrum next, so I too would like MD support. I assume the PCB modification would require to buy a new one?

**Comment by willyvmm** [comment]
> I got a spectrum next, so I too would like MD support. I assume the PCB modification would require to buy a new one?

Yes. SEGA pinout is different. 5v and GND are in different places.
An alternative may be a nasty Adapter.
